airflow/contrib/hooks/salesforce_hook.py

```diff
@@ -303,18 +281,12 @@ def write_object_to_file(
         #   (as is fairly standard for JavaScript)
         #   And for csv, we do a string
         if fmt == "csv":
-            # there are also a ton of newline objects
-            # that mess up our ability to write to csv
+            # there are also a ton of newline objects that mess up our ability to write to csv
             # we remove these newlines so that the output is a valid CSV format
             self.log.info("Cleaning data and writing to CSV")
             possible_strings = df.columns[df.dtypes == "object"]
-            df[possible_strings] = df[possible_strings].apply(
-                lambda x: x.str.replace("\r\n", "")
-            )
-            df[possible_strings] = df[possible_strings].apply(
-                lambda x: x.str.replace("\n", "")
-            )
-
+            df[possible_strings] = df[possible_strings].apply(lambda x: x.str.replace("\r\n", ""))
+            df[possible_strings] = df[possible_strings].apply(lambda x: x.str.replace("\n", ""))
```

Comment:
Could we use single line to remove rather than L288-L289?

Reply:
Yes, thanks for pointing that out. 👍

Reply:
You welcome.